security/anyuid_rbac.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: opea-chatqna
+  namespace: opea-chatqna
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: opea-chatqna
+  namespace: opea-chatqna
+rules:
+- apiGroups:
+  - security.openshift.io
+  resources:
+  - securitycontextconstraints
+  resourceNames:
+  - anyuid
```

Comment:
@vbedida79  For the prototype release, I am ok to use anyuid. However, we should figure out the reason why "anyuid" needs to be used. Could we file an issue to track the task? Thanks!